docs/docs/guides/reverse_proxy.md

```diff
@@ -178,3 +179,32 @@ The above configuration will create a "service" in Traefik, automatically adding
 It will also add a router, routing requests to "traefik.example.com" to your local container.
 
 Note that with this approach, you don't need to expose any ports for the Frigate instance since all traffic will be routed over the internal Docker network.
+
+## Caddy Reverse Proxy
+This example shows a working example of frigate running under a subdomain with logging and a tls cert (in this case a wildcard domain cert obtained independently of caddy) handled via imports
```

Comment:
```suggestion
This example shows Frigate running under a subdomain with logging and a tls cert (in this case a wildcard domain cert obtained independently of caddy) handled via imports
```

Reply:
Hmm I thought I committed this suggestion and the one above it.

Reply:
Oh, I did. The github UI confused me.